adaboost/tests/test_cuda_data_structures.hpp

```diff
@@ -77,3 +77,45 @@ TEST(Cuda, MatrixGPU)
         }
     }, std::logic_error);
 }
+
+TEST(Cuda, MatricesGPU)
+{
+    adaboost::utils::cuda::cuda_event_t has_happened;
+    adaboost::utils::cuda::cuda_event_create(&has_happened);
+    adaboost::cuda::core::MatrixGPU<float> mat_f;
+    EXPECT_EQ(0, mat_f.get_cols())<<"Number of columns should be 0";
+    EXPECT_EQ(0, mat_f.get_rows())<<"Number of rows should be 0.";
+    adaboost::cuda::core::MatrixGPU<float> mat1(3, 3), mat2(3, 3), mat3(2, 1);
+    adaboost::cuda::core::fill(float(4.0),mat1,0,0);
```

Comment:
```suggestion
    adaboost::cuda::core::fill(float(4.0), mat1, 0, 0);
```
Add spaces after `,` in other places as well.

Reply:
ok, will update in other places as well